### PR TITLE
Limit recent call logs to 100 

### PR DESCRIPTION
**Pull request by Naveen3Singh**
As suggested [here](https://issuetracker.google.com/issues/175198972?pli=1#comment6), `Calls.LIMIT_PARAM_KEY` is the correct way to do this.

### CODE REVIEW

**Comment by tibbi** [comment]
have you tested it on lower OS versions too?

File: app/src/main/kotlin/com/simplemobiletools/dialer/helpers/RecentsHelper.kt [hunk 1]
```diff
@@ -44,7 +45,10 @@ class RecentsHelper(private val context: Context) {
         val contactsNumbersMap = HashMap<String, String>()
         val contactPhotosMap = HashMap<String, String>()
 
-        val uri = Calls.CONTENT_URI
+        val uri = Calls.CONTENT_URI.buildUpon()
+            .appendQueryParameter(Calls.LIMIT_PARAM_KEY, QUERY_LIMIT)
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/dialer/helpers/RecentsHelper.kt:49]: so this works well on Android 6 too?

Comment (Naveen3Singh) [reply app/src/main/kotlin/com/simplemobiletools/dialer/helpers/RecentsHelper.kt]: just fixed that in https://github.com/SimpleMobileTools/Simple-Dialer/pull/379/commits/d6025e59624641f016412e08bdbb6c7a64b11512

**Comment by tibbi** [comment]
alright then, lets hope it works :) I changed it to 200 because of grouping of subsequent calls from the same number etc. Thanks

**Pull request merged**